data/exclusions-disease-gene.tsv

```diff
@@ -1,8 +1 @@
-omim_id	mondo_id	mondo_label	orcid	exclusion_reason_comment
-OMIM:603956	MONDO:0002974	cervical cancer'	https://orcid.org/0000-0002-4142-7153	evidence of various genes involved
-OMIM:619151	MONDO:0030894	"AMED syndrome, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
-OMIM:158901	MONDO:0008031		https://orcid.org/0000-0002-4142-7153	digenic
-OMIM:108770	MONDO:0007171	atrial standstill 1'	https://orcid.org/0000-0002-4142-7153	digenic
-OMIM:620040	MONDO:0031057	"dyskeratosis congenita, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
-OMIM:619478	MONDO:0030355	"facioscapulohumeral muscular dystrophy 4, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
-OMIM:300818	MONDO:0010438	paroxysmal nocturnal hemoglobinuria 1	https://orcid.org/0000-0002-4142-7153	"disease caused by a somatic mutation, therefore a gene association stating this is due to a germline mutation should not be added"
\ No newline at end of file
+omim_id	mondo_id	mondo_label	orcid	comment
```

Comment:
@joeflack4 - why is this file maintained?

Reply:
We talked briefly about that but I wasn't 100% clear either. Maybe you were talking about something else, but it sounded like you thought we might as well keep the logic (and empty file) here in order to support explicit exclusions in the future, should we ever need. And this way if we do that, I don't have to re-add all the logic. And I agree with this approach.

Reply:
No, that was not my intent when we discussed this. However, there is a comment from Sabrina in the Update OMIM PR [here](https://github.com/monarch-initiative/mondo/pull/8624#discussion_r1936458946) where the keeping this might be relevant depending on how you want to implement this. Basically, there are also some gene associations that match the rule, but for a curator reading the description they will know that the gene association should not be added. In these cases, the gene association in Mondo should not be added.

Reply:
Also RE:
> See comments inline. Which ticket is this?

There is no ticket for this. This is just what we discussed on call today to address issues in:
- https://github.com/monarch-initiative/mondo/pull/8624

Reply:
I read [the thread](https://github.com/monarch-initiative/mondo/pull/8624#discussion_r1936638034). Well it is good that we are keeping the exclusions logic, then. I think all we need to do is add an entry for it in [`exclusions-disease-gene.tsv`](https://github.com/monarch-initiative/omim/blob/main/data/exclusions-disease-gene.tsv).

I'm writing the docs for that momentarily. I'll let you respond and link it to her unless you want me to.

Let me know if you want me to add that entry to the exclusions file.

Reply:
FYI I plan to also run a `mondo-ingest` build tonight.

Reply:
Never mind. I forgot the outputs for this that get processed by `mondo` are in the OMIM release, not `mondo-ingest`. So I don't need to run the `mondo-ingest` build.

I haven't thoroughly looked at the outputs yet. Just the example described in detail in this PR. Trish, if you want to check when you have time, that'd be great. I can check too.